controllers/cloud.redhat.com/providers/sidecar/default.go

```diff
@@ -164,5 +176,9 @@ func getOtelCollector() *core.Container {
 			"memory": resource.MustParse("1024Mi"),
 		},
 	}
+	cont.VolumeMounts = []core.VolumeMount{{
+		Name:      fmt.Sprintf("%s-otel-config", appName),
+		MountPath: "/etc/otelcol/config",
```

Comment:
This path should be `/etc/otelcol/config.yaml`.

Reply:
thx, updated :)